docs/defining-models.md

```diff
@@ -1,11 +1,10 @@
 # Toucan Models
 
-In Toucan, you define the various *models* used in your application with `defmodel`.
-Models encapsulate information and behaviors related to a specific table in the application DB,
-and have their own unique record type.
+In Toucan, you define the various *models* used in your application with `defmodel`. Models encapsulate information and
+behaviors related to a specific table in the application DB, and have their own unique record type.
 
-Generally, a single Toucan model corresponds to a single table in your database. Common models might
-be things like `User` or `Venue`.
+Generally, a single Toucan model corresponds to a single table in your database. Common models might be things like
+`User` or `Venue`.
```

Comment:
what's the reason behind changing these lines? 

it makes it a lot harder to review PRs when things are randomly changed like this for no reason